buildscripts/checkgopath.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 #
-# Minio Cloud Storage, (C) 2015, 2016 Minio, Inc.
+# Minio Cloud Storage, (C) 2014-2018 Minio, Inc.
```

Comment:
Its better to list years clearly than ranging.

Reply:
why is that? and why the above one not preferred? AFAIK both styles are preferred styles in Apache licensing.